spec/components/guardian_spec.rb

```diff
@@ -1389,32 +1389,51 @@
         expect(Guardian.new(post.user).can_edit?(post)).to be_truthy
       end
 
-      context 'post is older than post_edit_time_limit' do
-        let(:old_post) { build(:post, topic: topic, user: topic.user, created_at: 6.minutes.ago) }
+      describe 'post edit time limits' do
 
-        before do
-          SiteSetting.post_edit_time_limit = 5
-        end
+        shared_examples "edit old posts" do
```

Comment:
can we avoid shared_examples,  I think it just confuses stuff. makes the specs a bit harder to follow.